docs/docs/setup-vast/install/linux.md

```diff
@@ -0,0 +1,116 @@
+  <a class="button button--md button--info margin--md" href="https://github.com/tenzir/vast/releases/latest/download/vast-linux-static.tar.gz">Development</a>
+</div>
+
+You can also get static builds for a specific git commit. This involves
+navigating a bit through the github CI:
+
+1. Browse to the [VAST static workflow][vast-workflow]
+2. Click on the latest run, e.g., `Merge pull request...`
+3. Scroll to the end of the page
+4. Click on the artifact filename, e.g.,
+   `vast-v1.0.0-101-g6e7a4ef1a4-linux-static.tar.gz`
+
+[vast-workflow]: https://github.com/tenzir/vast/actions?query=branch%3Amaster+workflow%3A%22VAST%22
+
+## systemd
+
+VAST has native [systemd
+support](https://github.com/tenzir/vast/tree/master/systemd/) for running as
+system service. The service is sandboxed and runs with limited privileges.
+
+### Prepare the host system
+
+Please note that all subsequent commands require `root` privileges. The service
+requires a user and group called `vast`. You can create them as follows.
+
+```bash
+useradd --system --user-group vast
+```
+
+Make sure that you don't grant any special rights to this user, e.g., do not
+enable `sudo` or other privileged commands. Once the user exists, you should
+create the directory for VAST's persistent data and change the permissions such
+that it is owned by the new `vast` user:
+
+```bash
+mkdir -p /var/lib/vast
+chown -R vast:vast /var/lib/vast
+```
+
+The systemd unit passes a
+[vast.yaml](https://github.com/tenzir/vast/tree/master/systemd/) configuration
+file to the VAST process. Make sure that the new user can read the `vast.yaml`.
+
+### Configure the unit
+
+Before you begin, find the line beginning with `ExecStart=` at the very bottom
+of the `[Service]` section in the unit file. Depending on your installation path
+you might need to change the location of the `vast` binary and configuration
+file.
+
+```config
+ExecStart=/path/to/vast --config=/path/to/vast.yaml start
+```
+
+In case you plan to run a PCAP source directly inside the VAST server process
+via `vast spawn source pcap`, you need to make sure that the VAST process gets
+the required privileges to listen on the network interfaces.
+
+```ini
+CapabilityBoundingSet=CAP_NET_RAW
+AmbientCapabilities=CAP_NET_RAW
+```
+
+Then link the unit file to your systemd search path:
+
+```bash
+systemctl link vast.service
+```
+
+To have the service start up automatically on system boot, `enable` it via
+`systemd`. Otherwise, just `start` it to run it immediately.
+
+```bash
+systemctl enable vast
+systemctl start vast
+```
+
+## Distribution Support
+
+:::tip Community contributions wanted!
+We are striving to bring VAST into the package managers of all major Linux
+distributions. Unfortunately we can do so only at a best-effort basis, but
+we much appreciate community contributions.
+:::
+
+### Debian
```

Comment:
@satta I've lost track about the current issues we face regarding Debian packaging. Mind providing a brief update?

Reply:
Yes. From my point of view I am currently seeing the following roadblocks for providing up-to-date packages for VAST:

* Lack of packaging for VAST dependencies in the required versions, most importantly Apache Arrow. See this [Request For Package (ITP)](https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=970021). A preliminary package [exists](https://salsa.debian.org/science-team/arrow) but needs stable maintainers who are willing and able to follow upstream's fast development process and ensure that new breaking releases are dealt with in a Debian policy-compliant way.
* (Minor issue) Lack of support for some (most?) non-Intel architectures that Debian supports. Not a dealbreaker but a nice-to-have.
* Missing ABI stability in VAST to provide a proper shared library package plus development headers package. Each ABI break would mean a new ABI version bump -- this essentially means that each ABI change would cause the new package to be treated like a completely new and unreviewed package that has to be manually cleared. Moreover one would need to ensure that the VAST version always matches the libvast version since IIRC there is no backward compatibility.

Reply:
The third bullet would essentially require us to define a stable subset of libvast that we offer as public API with a stable ABI, but that adds impediments that no sane startup wants to deal with if they don't need to. I don't think there'll be an agreement between us and the Debian way of doing things in this regard for years to come.

We should probably just erase the section here from the docs page.

Reply:
Thanks for laying it out, @satta. I think (1) and (2) are possible to overcome.

(3) is complicated. ABI stability for libvast is not feasible for us, as @dominiklohmann pointed out. 

That said, our *plugin API* is a stable point of reference. I'm not sure if Debian requires true ABI stability or whether a subset suffices. In any case, our main dependency Arrow is also moving fast, without ABI guarantees, so the problem also applies to (1).

If ABI stability is not negotiable, we probably have to create our own repository where we simply serve DEBs for libvast, heeaders, and plugins. That said, before considering "our own", we probably would try to find out if [OBS](https://openbuildservice.org/) suits the bill.